### PR TITLE
Proofreading

### PR DESCRIPTION
**Pull request by notque**
May or may not be helpful, add or don't add. Makes more sense read like this to me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,4 +46,4 @@ would find it with the following configuration snippet:
 Server = file:///path/to/output
 ```
 
-ART keeps a cache file (`.art-cache`) in its current working directory to speed up incremental rebuilds.
+ART keeps a cache file (`.art-cache`) in it's current working directory to speed up incremental rebuilds.
```
Comment (majewsky) [inline README.md:49]: Was correct before.

File: README.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ For each source in this configuration file, the following globs will be expanded
 * `$SOURCE_PATH/*/PKGBUILD` will be built with [makepkg(8)](https://www.archlinux.org/pacman/makepkg.8.html).
 * `$SOURCE_PATH/*.pkg.toml` will be built with [holo-build(8)](https://github.com/holocm/holo-build).
 
-Note that directories will not be traversed any deeper than that. For example, any PKGBUILD file must be in a direct
-subdirectory of the source path specified in the configuration file.
+All desired PKGBUILD files must be in a direct subdirectory of the source path specified in the configuration file. 
+Directories will not be traversed any deeper than that.
```
Comment (majewsky) [inline README.md:38]: I think this is backwards now. The change puts the emphasis on the concrete case of PKGBUILDs, while the previous wording refers to both globs equally and uses PKGBUILDs only as an example.

Comment (notque) [reply README.md:38]: Ah, Fair enough.

File: README.md (hunk 1 above)
Comment (notque) [reply README.md:49]: Don't understand your comment.

Comment (majewsky) [reply README.md:49]: What I mean is that `its` is correct here. When you expand `it's` to `it is` in your mind, it should be clearer that it doesn't make sense that way.

Comment (notque) [reply README.md:49]: Oh, grammar. GG German teaching me proper English grammar.

**Comment by majewsky** [comment]
This leaves only the "just" and "thus" to be deleted. AFAICS these are not wrong, but a matter of personal style, so I'm closing this. Thanks for taking a closer look, however! It appears that I have found the right timeframe for releases that maximizes your engagement with the release. :)

**Pull request closed without merging**